hip-clang.docker

```diff
@@ -19,6 +19,7 @@ RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y --allow-
     gdb \
     git \
     lcov \
+    libhiredis-dev \
```

Comment:
I would prefer not to require installing this package. Perhaps we could either us an option to disable it with ccache(if its available) or install the ccache package instead(as long as its version greater than 4 on ubuntu).